transcom/mymove
# Update logic for how button is rendered

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16066)

## Summary

This ensure that only one button is shown for users that have multi roles. 

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the office app as a TIO to find a move that has a status of payment requested and make note of the move locator code. 
2. Using that code login as TIO and you should see the button enabled (you may have to review the weights)
3. Login as the TOO and find the same move using the move code. You should see that the button is disabled.
4. Login as a multi user and find the same move using the move code. You should only see one instance of the button and it should appear in the same state as the TIO.


## Screenshots
![image](https://github.com/transcom/mymove/assets/84742904/942cd148-111f-4926-804b-cfde7359c81a)
